koji-builder/bin/koji-setup.sh

```diff
@@ -46,9 +46,12 @@ check_result 'Package docker-hello-world already exists in tag dest' \
 check_result 'Package docker-hello-world-source already exists in tag dest' \
              koji add-pkg dest docker-hello-world-source --owner kojiadmin
 
-check_result 'btype already exists' \
+check_result 'btype operator-manifests already exists' \
```

Comment:
Did the format of the error message change?

Reply:
No sorry, I'm just retarded. Fixed